src/en/charms-fan-aws-vpc.md

```diff
@@ -0,0 +1,65 @@
+Title: Creating an AWS VPC
+
+# Creating an AWS VPC
+
+This is in connection with [Juju and FAN networking][charms-fan] where an AWS
+user may wish to create an additional VPC (virtual private network).
+
+The procedure covered here refers to the AWS web interface. Presumably it is
+possible to do the same via the AWS API.
+
+To add a VPC:
+
+ 1. Navigate to the VPC console:
+
+    https://console.aws.amazon.com/vpc/home
+
+ 1. **Create a VPC** by selecting `Your VPCs` in the left menu and pressing the
+    `Create VPC` button.
+
+    In the resulting dialog box, enter a name for the VPC (e.g. 'vpc-juju') and
+    a /16 address space (e.g. '192.168.0.0/16').
+
+ 1. **Create a subnet** for the VPC by selecting `Subnets` in the left menu and
+    pressing the `Create Subnet` button.
+
+    In the resulting dialog box, enter a name for the subnet (e.g.
+    'vpc-juju-subnet'), select the VPC you created earlier, an Availability
+    Zone (optional), and an IPv4 CIDR block that resides within the initial
+    address space (e.g. '192.168.1.0/24').
+
+    Select the subnet's checkbox and hit `Subnet Actions`. There, choose
+    `Modify auto-assign IP settings` and then select `Enable auto-assign public
+    IPv4 address`.
+
```

Comment:
For complete HA - create one subnet for each Availability Zone.

Reply:
OK

Reply:
Actually, I don't quite get this comment. HA also requires manipulation of the applications. Unless I'm missing something, I'd rather not get into that.